pkg/distro/rhel9/imagetype.go

```diff
@@ -383,16 +380,20 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 	}
 
 	mountpoints := customizations.GetFilesystems()
-
-	if mountpoints != nil && t.rpmOstree {
-		return warnings, fmt.Errorf("Custom mountpoints are not supported for ostree types")
-	}
-
 	err := blueprint.CheckMountpointsPolicy(mountpoints, pathpolicy.MountpointPolicies)
 	if err != nil {
 		return warnings, err
 	}
 
+	if mountpoints != nil && t.rpmOstree && t.name != "edge-raw-image" {
```

Comment:
I know, but as I mentioned, it is too late, because the code can return with FS customization related error on line #385 for ostree image which does not support FS customizations at all, which would be IMO confusing.

Reply:
Changed the ordering to return early incase fs customization not allowed